### PR TITLE
Added MOAC testnet, mainnet and icon

### PR DESCRIPTION
**Pull request by mdranger**
Please let me know if you have any questions

### CODE REVIEW

File: _data/chains/eip155-201.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  },
+  "infoURL": "https://moac.io",
+  "shortName": "moac",
+  "chainId": 201,
+  "networkId": 201,
+  "explorers": [
+    {
+      "name": "moac testnet explorer",
+      "url": "https://testnet.moac.io",
+      "standard": "EIP3091"
```
Comment (ligi) [inline _data/chains/eip155-201.json:19]:
```suggestion
      "standard": "none"
```

the explorer does not respect EIP3091 when it comes to addresses

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
please apply my suggestions of fix it yourself

**Review by mdranger — COMMENTED** [review]
updated

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `MOAC` more than once

**Review by mdranger — COMMENTED** [review]
change the testnet shortName to moactest, mainnet shortName uses moac

**Review by ligi — APPROVED** [review]

**Pull request merged**